crypto/aes/aes_wrap.c

```diff
@@ -21,13 +21,13 @@ int AES_wrap_key(AES_KEY *key, const unsigned char *iv,
                  unsigned char *out,
                  const unsigned char *in, unsigned int inlen)
 {
-    return CRYPTO_128_wrap(key, iv, out, in, inlen, (block128_f) AES_encrypt);
+    return (int)CRYPTO_128_wrap(key, iv, out, in, inlen, (block128_f) AES_encrypt);
```

Comment:
It looks like the bounds check in CRYPTO_128_wrap() is wrong..
it returns inlen +8 but checks for inlen > (1<<31)

Reply:
In what realistic scenario do we care about _key wrap_ being called on data this big?